### PR TITLE
Create deploymkdocs.yml

### PR DESCRIPTION
**Pull request by mikemaddem**

### CODE REVIEW

File: .github/workflows/deploymkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
```
Comment (techlover1) [inline .github/workflows/deploymkdocs.yml:10]: Don't deploy on pr

File: .github/workflows/deploymkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
```
Comment (techlover1) [inline .github/workflows/deploymkdocs.yml:3]: Call it deploy or something idk

**Review by techlover1 — CHANGES_REQUESTED** [review]

**Review by techlover1 — APPROVED** [review]

**Review by cokenyo — APPROVED** [review]
MMMM :)

**Review by loganramhap — APPROVED** [review]
Perfect <3 mike

**Review by kennedfe — COMMENTED** [review]
epic

**Pull request merged**